x/staketia/keeper/unbonding.go

```diff
@@ -29,6 +35,12 @@ func (k Keeper) RedeemStake(ctx sdk.Context, redeemer string, stTokenAmount sdkm
 		return nativeToken, err
 	}
 
+	// If the remaining delegated balance for staketia is 0, that means we've undelegated
+	// all the stake in the MS account and the redemptions should be switched over to stakeibc
+	if hostZone.RemainingDelegatedBalance.LTE(sdkmath.ZeroInt()) {
+		return nativeToken, types.ErrRedemptionsDisabled
```

Comment:
Where does `RedemptionsEnabled` get set to false?

Reply:
it's set to false in the upgrade handler